phpcs.xml

```diff
@@ -1,16 +1,17 @@
 <?xml version="1.0"?>
 <ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="vendor/squizlabs/php_codesniffer/phpcs.xsd">
 
+    <config name="php_version" value="70499"/>
```

Comment:
this shouldn't be necessary as we've changed the CI matrix so that QA tooling (such as codestyle checks) are only executed on the minimum supported PHP version of a component.

As this component requires PHP 7.4 as its minimum, you can remove this line here so it won't lead to issues when removing PHP 7.4.

Reply:
I see. I guess this happened since I'm running php 8 locally and stumbled upon this.

Reply:
Kept this: still useful for local runs